src/main/java/epam/gym/service/TrainerService.java

```diff
@@ -1,10 +1,31 @@
 package epam.gym.service;
 
 import epam.gym.entity.Trainer;
+import epam.gym.entity.Training;
+
+import javax.naming.AuthenticationException;
```

Comment:
I think we can create our own exception AuthenticationException, which extends RunTimeException. And not to add throws AuthenticationException everywhere after method

Reply:
added own exception AuthenticationException, deleted throws AuthenticationException after methods